src/command_modules/azure-cli-dla/azure/cli/command_modules/dla/_help.py

```diff
@@ -25,6 +25,11 @@
 helps['dla account blob-storage add'] = """
 type: command
 short-summary: Links an Azure Storage account to the specified Data Lake Analytics account.
+examples:
+  - name: Links an Azure Storage account to the specified Data Lake Analytics account. (autogenerated)
+    text: |-
```

Comment:
I know this is a new example. But do you know why this is '|-' instead of '|'? Does it do anything special?

Reply:
That was because the example doesn't have a trailing '\n'. Is it okay to still have these?

Reply:
Does the hyphen add a trailing newline?

Reply:
The hyphen tells yaml loader to not put a newline at the end of the loaded string.